llama.cpp

```diff
@@ -2155,13 +2154,11 @@ llama_token llama_sample_token_mirostat(struct llama_context * ctx, llama_token_
 
     if (ctx) {
         ctx->t_sample_us += ggml_time_us() - t_start_sample_us;
-        ctx->n_sample++;
```

Comment:
I think this change is incorrect. `n_sample` is incremented in the other sampling paths except for mirostat v2 (but to me this seems like mirostat v2 is incorrect).

Reply:
llama_sample_token will increase n_sample.

Reply:
It seems you're right.